.gitignore

```diff
@@ -5,3 +5,4 @@ cmd/VERSION
 *.swp
 *.swo
 .idea
+/pebble
```

Comment:
Curious.. what's this about?

Reply:
It's because I do `go build ./cmd/pebble` to build locally, and that produces a `pebble` binary in the current directory. It was odd to me that you need to prefix with `/` (looks like an absolute path), but that's `.gitignore` syntax for "only the file `pebble` in the root repository directory, not `pebble/foo` or `bar/pebble`".

Reply:
I see.. I generally have cmd/foo/foo in .gitignore, since that's what we get if we run "go build" there. Otherwise it'll drop wherever $CWD happens to be.

Either way, if that helps you somehow, that's certainly okay.